pkg/tests/end_to_end_tests/migrate_test.go

```diff
@@ -146,6 +139,8 @@ func verifyExtensionExists(t *testing.T, db *pgxpool.Pool, name string, expectEx
 }
 
 func TestInstallFlagPromscaleExtension(t *testing.T) {
+	// TODO (james): remove this skip
```

Comment:
This is being tracked here: https://github.com/timescale/promscale_extension/issues/95